### ISSUE TITLE
Suggestion - Collapse information into one menu option [example design included]

### ISSUE THREAD

**Issue by UnIockables** [microtavor5/PetInfoPlugin#12]
Hey Micro, thanks for this plugin! I mainly use this to see who owns which pet in busy areas, but the info is a really nice touch too!
I had an idea on how to simplify the plugin, and thought I'd share the idea with you.

![petinfo_redesign](https://user-images.githubusercontent.com/68065008/128008262-36c5e724-487d-42e6-8521-38fc6729a48f.png)
_Replace `<Player>` with the owner's IGN, and `<Pet>` with the type of pet it is. Clicking this would function as it currently does by describing where/how to obtain that pet._ 

The benefit of this is you'd be able to see the owner without ever needing to print that information to the chat box, and it would also remove one of the menu options, which would reduce screen clutter.

---
Alternatively, can I request an option in the plugin settings to disable the Info menu option? 
(Obviously I'd prefer my suggestion rather than this approach, but if for whatever reason it is not possible, a smaller right click menu is my main goal here and I mainly use this plugin for checking owners.)

Thanks for reading, hope you have a great day 😊

**Comment by microtavor5** [comment]
Thanks for the suggestion! Not sure why I didn't think to do it myself. I made the plugin mostly for the info, so that's the paradigm I used when making it. I had the thought "why not add the owner info too, that would be fun", and have since then has it mentioned a few times that the plugin is useful for that feature alone.
The combination actually would make things simpler on the back end too. Long story short, I'll add it to the list! I'm a little busy right now but I plan on getting back to the plugin in the next little bit (a few weeks max) so if you don't see any movement at that point, feel free to guilt me into working ;)

**Comment by microtavor5** [comment]
I've implemented the change, but I wanted to know from your perspective on the coloring of the owner's name to match their relative combat strength. Is it annoying? Should I add the option to make it the default yellow instead? tbh I'm probably over thinking this but worth asking

**Comment by UnIockables** [comment]
Wow, awesome that you did it so fast! I was just happy to see you liked the idea and would consider it. Thanks for following it up 😁
As for the Owner's colouring, I honestly didn't even realise that it was getting it's colour from their combat level until you mentioned it there; in the time I'd used the plugin I only noticed it being green (so for the above example I just copied what I saw there). Definitely not an annoying feature but one that I guess wasn't exactly clear what it represented. 

I'd personally just make the player name white to match the other player right-click options, but that's just me. Would a colour picker in the plugin settings for the Owner's name be possible, or is that too much work?
